### DISCUSSION TITLE
How to add CSS or JS files to my custom theme?

### DISCUSSION THREAD

**Discussion by Danieleeffe1** [OpenMage/magento-lts#3508] [Questions & Answers]
Sorry guys, but I'm trying to add a custom css file to my Mimic theme

With Magento it was enough to add a local.xml file in the theme's layout folder, but now this works no longer.

how do you add a css or js file into a custom theme?

**Comment by addison74** [comment]
It doesn't work anymore only if we broke something in the meantime. I will check, although I have one OpenMage in production with a local.xml file that loads two JS files in the Backend.

**Reply by Danieleeffe1** [reply]
strange...

all the examples I found online don't work.

Can you please send an example that loads a css file or a js file?

Thank you.

**Reply by kiatng** [reply]
If this is related to your mimic theme, why don't you create an issue in your repo and see if someone can help you there.

**Reply by Danieleeffe1** [reply]
this is a sample xml I found that doesn't work

https://gist.github.com/anthuanvasquez/72d150674e4c0bf11bc8

**Reply by Danieleeffe1** [reply]
I tried setting a local.xml file in rwd

the result does not change

it does not load css files and makes no changes

**Reply by Danieleeffe1** [reply]
I currently reinstalled OpenMage from scratch version 20.1.1 clean install

**Reply by Danieleeffe1** [reply]
ok....

After several attempts I succeeded.

At this point I don't understand why the same local.xml file in magento works and in openmage doesn't work .

It does not matter
Thanks for the support and sorry for the inconvenience.
